[2024-06-24T11:40:38.203+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:40:38.243+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_odesa scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:40:38.264+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_odesa scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:40:38.265+0000] {taskinstance.py:2306} INFO - Starting attempt 5 of 5
[2024-06-24T11:40:38.293+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): processing_tasks.inject_data_odesa> on 2024-06-22 00:00:00+00:00
[2024-06-24T11:40:38.307+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1741) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T11:40:38.309+0000] {standard_task_runner.py:63} INFO - Started process 1761 to run task
[2024-06-24T11:40:38.311+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.inject_data_odesa', 'scheduled__2024-06-22T00:00:00+00:00', '--job-id', '181', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpslfoxikv']
[2024-06-24T11:40:38.315+0000] {standard_task_runner.py:91} INFO - Job 181: Subtask processing_tasks.inject_data_odesa
[2024-06-24T11:40:38.406+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.inject_data_odesa scheduled__2024-06-22T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T11:40:38.719+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.inject_data_odesa' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-22T00:00:00+00:00'
[2024-06-24T11:40:38.724+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:40:38.724+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719014400), 
                   292.8,
                   60,
                   0,
                   3.05,
                   'Odesa'
            );
[2024-06-24T11:40:38.741+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:40:38.765+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:40:38.773+0000] {sql.py:487} INFO - Running statement: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719014400), 
                   292.8,
                   60,
                   0,
                   3.05,
                   'Odesa'
            );, parameters: None
[2024-06-24T11:40:38.775+0000] {sql.py:496} INFO - Rows affected: 1
[2024-06-24T11:40:38.778+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:40:38.796+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.inject_data_odesa, run_id=scheduled__2024-06-22T00:00:00+00:00, execution_date=20240622T000000, start_date=20240624T114038, end_date=20240624T114038
[2024-06-24T11:40:38.850+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:40:38.869+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
